receiver/hostmetricsreceiver/internal/scraper/memoryscraper/metadata.yaml

```
@@ -28,3 +28,12 @@ metrics:
     gauge:
       value_type: double
     attributes: [state]
+
+  system.linux.memory.available:
+    enabled: false
+    description: Linux-specific more accurate alternative to system.memory.usage with state=free
```

```suggestion
    description: An estimate of how much memory is available for starting new applications, without swapping. This is a more accurate alternative than system.memory.usage with state=free. (Linux only)
```
It might be good to have the actual description of what this metric is here, instead of simply a comparison to a different metric. I realize this might be a bit verbose so feel free to modify. Just a suggestion.

If it's obvious to users what this is for, feel free to ignore.